Public/Src/Deployment/privatePackages.dsc

```diff
@@ -94,6 +95,17 @@ namespace PrivatePackages {
                         importFrom("BuildXL.Utilities.Instrumentation").withQualifier(winx64Qualifier).Common.dll.runtime,
                     ],
                 },
+                {
+                    subfolder: r`lib/netcoreapp3.0/osx-x64`,
+                    contents: [
+                        importFrom("BuildXL.Utilities").withQualifier(osxx64Qualifier).dll.runtime,

```

Comment:
Putting the common subset into a function and calling it with the qualifier would reduce duplication / potential errors. The whole cache reusable dll business is malformed (we have to fix this later), as we should just publish the old assemblies .NET and one assembly for .NET Core, which is targeting .NETStandard2+.